leo_bringup/config/imu_filter.yaml

```
@@ -7,3 +7,7 @@
     orientation_variance: 0.001
     bias_save_period: 120
     do_save_bias: true
+    angular_velocity_threshold: 0.15
+    acceleration_threshold: 0.1
+    delta_angular_velocity_threshold: 0.01
+    required_steady_time: 1.0
```

Let's rename these parameters to indicate they are related to steady state estimation:
```suggestion
    steady_state_angular_velocity_threshold: 0.15
    steady_state_acceleration_threshold: 0.1
    steady_state_delta_angular_velocity_threshold: 0.01
    steady_state_required_time: 1.0
```